docs/UserGuide.md

```diff
@@ -44,9 +44,18 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
   - `delete-lecture /module {module_code} /lecture {lecture_id}`: Deletes the specified lecture from the specified module
   - `delete-video /module {module_code} /lecture {lecture_id} /video {video_id}`: Deletes the specified video from the specified lecture from the specified module
 - Tag
-  - `tag /module {module_code} /lecture {lecture_id} /description {tag_description}`: Tags a module from Le Tracker
-  - `untag /module {module_code} /lecture {lecture_id} /tag {tag_id}`: Untags a module from Le Tracker
-
+  - `tag {module_code} /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`: Tags a module from Le Tracker
+  - `tag {lecture_name} [/mod {module_code}] /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`: Tags a lecture from 
+    a module 
+  - `tag {video_name} [/lec {lecture_name}] [/mod {module_code}] /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`: 
+    Tags a video from a lecture
+  - `untag {module_code} /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`: Removes specified tags from a module 
+    from Le Tracker
+  - `untag {lecture_name} [/mod {module_code}] /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`: Removes the 
+    specified tags from a lecture 
+  - `untag {video_name} [/lec {lecture_name}] [/mod {module_code}] /tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]`:
+       Removes the specified tags of a video
```

Comment:
Think you have some typos here: `/tags {tag 1} {tag_1}[, {tag_2}[, {tag_3}, ...]]]` should probably be /tags {tag_1}[, {tag_2}[, {tag_3}, ...]]]